src/board.h

```diff
@@ -0,0 +1,90 @@
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "block.h"
+
+typedef unsigned int uint;
+
+enum MoveDirection
+{
+    LEFT,
+    DOWN,
+    RIGHT,
+    UP
+};
+
+extern constexpr char moveRow[4] = {0, +1, 0, -1};
+extern constexpr char moveCol[4] = {-1, 0, +1, 0};    // imo nie ma sensu tego trzymać w strukturze pozycji
+
+struct Position
+{
+    int row, col;
```

Comment:
To nie jest ogólna pozycja, więc niech się nazywa BoardPosition (bo jest row i col, a nie np. x i y)